docs/guides/8-continuous-integration.md

```diff
@@ -47,6 +47,30 @@ Learn more about [CircleCI Configuration](https://circleci.com/docs/2.0/config-i
 
 Spectral has a pre-built [Spectral GitHub Action](https://github.com/stoplightio/spectral-action) which should speed up implementing Spectral in your GitHub repository.
 
+## GitLab
+
+GitLab users can add the following to their `.gitlab-ci.yml` files:
+
+```yaml
+stages:
+  - lint
+
+lint:spectral:
+  stage: lint
+  image:
+    name: stoplight/spectral
+    entrypoint: [""]
+  script:
+    - spectral lint -D -f junit -o spectral-report.xml openapi.yaml
+  artifacts:
+    when: always
+    reports:
+      junit: $CI_PROJECT_DIR/spectral-report.xml
+      expire_in: 2 weeks
```

Comment:
I'm not very familiar with GitLab CI/CD but I tried to copy/paste this script and it was throwing an error for `expire_in`. I checked [their docs](https://docs.gitlab.com/ee/ci/yaml/index.html#artifactsexpire_in) and it looks like that property needs to be under `artifacts` instead of `reports` maybe?